package.json

```diff
@@ -16,7 +16,7 @@
     "hexo-generator-feed": "^1.1.0",
     "hexo-generator-tag": "^0.2.0",
     "hexo-renderer-ejs": "^0.2.0",
-    "hexo-renderer-marked": "^0.2.10",
+    "hexo-renderer-marked": "ngokevin/hexo-renderer-marked#9497bdd",
```

Comment:
tested it, and worked for me when changing `markdown` to `marked` (which you noticed/fixed in the other PR):

```diff
diff --git a/_config.yml b/_config.yml
index 30fe6a4..25b6331 100644
--- a/_config.yml
+++ b/_config.yml
@@ -93,7 +93,8 @@ feed:
 
 # Markdown
 ## https://github.com/chjj/marked
-markdown:
+## `modifyAnchors: 1` forces lowercasing of generated anchors for headings
+marked:
   gfm: true
   pedantic: false
   sanitize: false
@@ -101,6 +102,7 @@ markdown:
   breaks: true
   smartLists: true
   smartypants: true
+  modifyAnchors: 1
 
 # Deployment
 ## Docs: https://hexo.io/docs/deployment.html
diff --git a/package.json b/package.json
index 24a5b37..017641d 100644
--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "hexo-generator-feed": "^1.1.0",
     "hexo-generator-tag": "^0.2.0",
     "hexo-renderer-ejs": "^0.2.0",
-    "hexo-renderer-marked": "ngokevin/hexo-renderer-marked#9497bdd",
+    "hexo-renderer-marked": "github:pashokus/hexo-renderer-marked#0267532",
     "hexo-renderer-stylus": "^0.3.1",
     "hexo-server": "^0.2.0",
     "js-yaml": "^3.5.5",
```